src/components/active-preset/active-preset.js

```diff
@@ -16,6 +16,7 @@ const mapStateToProps = state => ({
     shell: state.shell,
     engine: {
       adBreak: state.engine.adBreak,
+      adIsBumper: state.engine.adIsBumper,
```

Comment:
bumper is not ad. change to `isBumper `

Reply:
You don't say! :)
Yes, but it uses our ad framework (events, presets, classes and more) so I prefer that to keep this terminology.

Reply:
I actually agree with @yairans as we have two types of bumpers today - kaltura bumpers and ad bumpers which are treated as ads.
I think we should reserve the name of adBumper for future use of real ad bumpers an call this `isBumper`.